internal/apis/certmanager/types_issuer.go

```diff
@@ -232,6 +238,28 @@ type VaultAppRole struct {
 	SecretRef cmmeta.SecretKeySelector
 }
 
+// VaultKubernetesAuth is used to authenticate against Vault using a client
+// certificate stored in a Secret.
+type VaultClientCertificateAuth struct {
+	// The Vault mountPath here is the mount path to use when authenticating with
+	// Vault. For example, setting a value to `/v1/auth/foo`, will use the path
+	// `/v1/auth/foo/login` to authenticate with Vault. If unspecified, the
+	// default value "/v1/auth/cert" will be used.
+	// +optional
+	Path string
```

Comment:
While testing, I struggled multiple times finding out the correct syntax for the path. I think it would be great if we added a some validation logic that checks that the path either is empty or starts with the prefix `/v1/auth/`. WDYT?

Reply:
Sure!

As the same topic AFAIK also applies to Kubernetes auth: Should it be added there, too? If yes, in this PR or in another one?